wallet/txrules/rules.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// Package txrules provides transaction rules that should be followed by
+// transaction authors for wide mempool acceptance and quick mining.
+package txrules
+
+import (
+	"errors"
+
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+)
+
+// DefaultRelayFeePerKb is the default minimum relay fee policy for a mempool.
+const DefaultRelayFeePerKb btcutil.Amount = 1e3
+
+// IsDust determines whether a transaction output value and script length would
+// cause the output to be considered dust.  Transactions with dust outputs are
+// not standard and are rejected by mempools with default policies.
+func IsDust(amount btcutil.Amount, scriptSize int, relayFeePerKb btcutil.Amount) bool {
```

Comment:
Seems like this really ought to be using the code from `btcd` which isn't exported yet.  We need to get the policy code off into a policy package to make that feasible though.